std/process.d

```diff
@@ -2239,17 +2246,15 @@ $(LREF nativeShell).
 */
 @property string userShell() @safe
 {
-    version (Windows)      return environment.get("COMSPEC", "cmd.exe");
-    else version (Android) return environment.get("SHELL", "/system/bin/sh");
-    else version (Posix)   return environment.get("SHELL", "/bin/sh");
+    version (Windows)      return environment.get("COMSPEC", nativeShell);
+    else version (Posix)   return environment.get("SHELL", nativeShell);
```

Comment:
The doc comment above says it uses `nativeShell` and we're just duplicating this list if we don't.